variables.tf

```diff
@@ -0,0 +1,34 @@
+variable "API_KEY" {
+  description = "API key for To-do application"
+}
+
+variable "API_TOKEN" {
+  description = "API token for To-do application"
```

Comment:
Like your outputs, labelling some of these as `sensitive = true` would be good to reduce the risk of variables being outputted to logs